packages/rancher-backup/charts/README.md

```diff
@@ -0,0 +1,57 @@
+# Rancher-Backup  Chart
+
```

Comment:
There needs to be some reference to https://github.com/rancher/backup-restore-operator so that users can look at the GitHub repository that contains the actual CRDs / controller that is being used.